### PR TITLE
Add cookie support

### PR DESCRIPTION
**Pull request by daydiff**
It's resolve this issue https://github.com/Mashape/unirest-php/issues/35

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ build
 vendor
 composer.lock
 composer.phar
+/nbproject/private/
```
Comment (ahmadnassri) [inline .gitignore]: can you please remove this from the commit, as its specific to your IDE / ENV

File: src/Unirest/Request.php [hunk 2]
```diff
@@ -59,6 +60,18 @@ public static function defaultHeaders($headers)
     }
 
     /**
+     * Set a coockie file path for enabling coockie handling
+     *
+     * $cookieFile must be a correct path with permission to write to them
+     *
+     * @param string $cookieFile - path to file for saving coockie
+     */
+    public static function cookieFile($cookieFile)
```
Comment (ahmadnassri) [inline src/Unirest/Request.php]: can you also update the README with reference to this? :)

**Comment by ahmadnassri** [comment]
great :+1: if you can look at my notes and update that'll be great, otherwise, I can check in again and take care of that myself.

thank you!

**Comment by ahmadnassri** [comment]
this needs rebasing, i might get to it later this weekend.

**Comment by cristianp6** [comment]
Please, could you merge this, I need this fix badly!
Thanks guys!

**Comment by ahmadnassri** [comment]
@cristianp6 i just made the fix, waiting for tests then releasing a new version

**Comment by ahmadnassri** [comment]
@cristianp6 @daydiff this is now available in [`v2.4.0`](https://github.com/Mashape/unirest-php/releases/tag/v2.4.0)

**Comment by cristianp6** [comment]
Thanks @ahmadnassri 
I've made a pull request https://github.com/Mashape/unirest-php/pull/77
hope it helps

**Comment by ahmadnassri** [comment]
ah, that's what happens when attempting to fix bad merges late at night (and when no unit-tests are written!) totally my bad! thanks @cristianp6 for fixing!

**Pull request closed without merging**